did/resolver/resolver_test.go

```diff
@@ -0,0 +1,44 @@
+package resolver
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDIDDocumentMetadata(t *testing.T) {
+	// good
+	var metadata DocumentMetadata
+	assert.True(t, metadata.IsValid())
+
+	// bad
+	badMetadata := DocumentMetadata{
+		Created: "bad",
+		Updated: time.Now().UTC().Format(time.RFC3339),
+	}
+	assert.False(t, badMetadata.IsValid())
+}
```

Comment:
Can't tell if this was existing, but the tests could be separated 
```suggestion
func TestDIDDocumentMetadata_IsValid(t *testing.T) {
	t.Run("returns true with empty", func(t *testing.T) {
        	var metadata DocumentMetadata
                assert.True(t, metadata.IsValid())
	})


	t.Run("returns false when created field is not a timestamp", func(t *testing.T) {
	  badMetadata := DocumentMetadata{
		Created: "bad",
		Updated: time.Now().UTC().Format(time.RFC3339),
	  }
	  assert.False(t, badMetadata.IsValid())
	})
}
```